scripts/Tools/code_checker

```diff
@@ -51,7 +51,6 @@ formatter_class=argparse.ArgumentDefaultsHelpFormatter
 def run_pylint(on_file):
 ###############################################################################
     pylint = find_executable("pylint")
-    expect(pylint is not None,"pylint not found")
```

Comment:
expect here was just causing a hang in scripts_regression_tests, moving it to main solved the problem